README.md

```diff
@@ -41,7 +41,7 @@ This is quite slow and can take up to one or two hours but can provide good resu
 
 -   Go
 -   PHP
--   TypeScript
+-   TypeScript, TSX (only type declaration using object type literal is counted for 'classes' metric, but not type alias.)
```

Comment:
I think the description in the parenthesis would be redundant because #203 is merged already.